spec/lib/pact/pact_broker/pact_selection_description_spec.rb

```diff
@@ -18,6 +18,12 @@ module PactBroker
         subject { pact_selection_description(provider, consumer_version_selectors, options, broker_base_url) }
 
         it { is_expected.to eq "Fetching pacts for Bar from http://broker with the selection criteria: latest for tag cmaster (or master if not found), all for tag prod, work in progress pacts created after 2020-01-01" }
```

Comment:
What do you think about splitting this test into more cases? I created new block to avoid making this text even longer. The same approach can be used to split it even further, like have a separate scenario for WIP option.